src/app/pages/transaction-request/components/transaction-error/error-messages.tsx

```diff
@@ -116,44 +118,23 @@ export const IncorrectContractAddressMessage = memo(props => {
   );
 });
 
-export const UnauthorizedErrorMessage = memo(props => {
-  useScrollLock(true);
+export const UnauthorizedErrorMessage = memo(() => {
   return (
-    <Fade in>
-      {styles => (
-        <Flex
-          position="absolute"
-          width="100%"
-          height="100vh"
-          zIndex={99}
-          left={0}
-          top={0}
-          alignItems="center"
-          justifyContent="center"
-          p="loose"
-          bg="rgba(0,0,0,0.35)"
-          backdropFilter="blur(10px)"
-          style={styles}
-        >
-          <ErrorMessage
-            title="Unauthorized request"
-            body="The transaction request was not properly authorized by any of your accounts. If you've logged in to this app before, then you might need to re-authenticate into this application before attempting to sign a transaction with the Hiro Wallet."
-            border={'1px solid'}
-            borderColor={color('border')}
-            boxShadow="high"
-            css={{
-              '& > *': {
-                pointerEvents: 'all',
-              },
-            }}
-            {...props}
-          />
-        </Flex>
-      )}
-    </Fade>
+    <Navigate
+      to={RouteUrls.Error}
+      state={{
+        body: `The transaction request was not properly authorized by any of your Hiro Wallet accounts. This typically happens if you've logged into this app before using another account.`,
+        helpText: [
+          `Please sign out of the app and sign back in to re-authenticate into the application. This
+          should help you successfully sign your transaction with the Hiro Wallet.`,
+        ],
+        title: 'Unauthorized request',
+      }}
+    />
```

Comment:
What's the benefit navigating to a generic error route?

I guess it saves having a new error route, _but_ sometimes that's helpful (analytics), and we can't pass non-primatives (components into the layout etc).

Could this component contain `<ErrorLayout />`, and have an associated custom route?  This will still be easy to reuse for the ledger error.

Reply:
Same, will refactor.